.github/workflows/sync.yml

```diff
@@ -0,0 +1,92 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: GitHub user
+      run: |
+        # https://api.github.com/users/github-actions[bot]
+        git config user.name 'github-actions[bot]'
+        git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+    - name: Fetch latest spec to oas.fetched/
+      id: fetch
+      run: |
+        make fetch
+        git add oas3.fetched
+        echo `git commit -m 'sync: fetch ${{ steps.date.outputs.date }} spec and apply patches'`
+    - name: Build single-file spec in oas3.stitched/metal_openapi.yaml
+      id: stitch
+      if: ${{ always() && steps.fetch.conclusion == 'success' }}
+      run: |
+        make stitch-spec
+        git add oas3.stitched
+        echo `git commit -m 'sync: stitch spec with ${{ steps.date.outputs.date }} spec'`
+    - name: Patch spec and build ./metal_openapi.fixed.yml
+      id: patch
+      if: ${{ always() && steps.stitch.conclusion == 'success' }}
+      run: |
+        make patch-spec
```

Comment:
Ah, right...I forgot that metal-python patches the stitched spec, not the split spec; in that case, it'll be a much bigger change to abandon the stitch step here.